apiserver/facades/agent/meterstatus/meterstatus.go

```diff
@@ -73,6 +85,18 @@ func NewMeterStatusAPI(
 	}, nil
 }
 
+// NewMeterStatusAPIWithState creates a new MeterStatusAPI, but uses the
+// state.State for creating the MeterStatusAPI. This is so we fulfill the
+// allfacades registration setup (i.e. it requires the state rather than
+// the well defined interface that the MeterStatusAPI requires)
+func NewMeterStatusAPIWithState(
```

Comment:
Can we please not use this facade constructor?
Instead please use this form:
```go
func NewMeterStatusFacade(ctx facade.Context) (*MeterStatusAPI, error) {
```

Reply:
@howbazaar of course :dancer: